core/modules/layout/includes/layout.layout.inc

```diff
@@ -76,6 +76,21 @@ function layout_layout_context_info() {
   return $info;
 }
 
+/**
+ * Implements hook_layout_relationship_info().
+ */
+function layout_layout_relationship_info() {
+  $info['user_from_node'] = array(
```

Comment:
Change to "Content: author" or "Content author" as per https://github.com/backdrop/backdrop-issues/issues/2134#issuecomment-754317337